timely_beliefs/beliefs/classes.py

```diff
@@ -276,18 +277,29 @@ def add_to_session(
         """
         # Belief timing is stored as the belief horizon rather than as the belief time
         belief_records = (
```

Comment:
Technically not "record" yet, as you do the `to_dict("records")` part later now. Still the `beliefs_data_frame`?

Reply:
Thanks for the suggestion